docs/browser/events.md

```diff
@@ -0,0 +1,166 @@
+# Semantic conventions for browser events
+
+**Status**: [Experimental](../../../document-status.md)
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [PageView](#pageview)
+- [PageNavigationTiming](#pagenavigationtiming)
+- [ResourceTiming](#resourcetiming)
+- [Exception](#exception)
```

Comment:
**Question:** This TOC item doesn't resolve. Where should this lead?

Wanted to look into this for the `componentStack` field that I would love to see :)

Reply:
It's either https://github.com/open-telemetry/semantic-conventions/pull/970 or something new;